prism-languages.php

```diff
@@ -4,6 +4,7 @@
  * Retrieve support languages
  * @return array Hash of lang => language
  */
```

Comment:
:no_entry_sign: **Error**: Missing @package tag in file comment (*Squiz.Commenting.FileComment.MissingPackageTag*).